### DISCUSSION TITLE
Show notification from Modal Compontent

### DISCUSSION THREAD

**Discussion by gallomaf** [usernotnull/tall-toasts#357] [Q&A]
I have this environment

I use Livewire Powergrid (https://livewire-powergrid.com) 
In each row, I have a button tha open a Modal ( https://github.com/wire-elements/modal ) 

In the modal there is a button for update the row of table.
Inside the update method of Modal Component, I have the following code:

`
public function update(){
 
        //change the state of spedizione (row of powergrid table)
        $this->spedizione->stato = $this->nuovo_stato;
        $this->spedizione->save();
       
        //use the toast notification
        toast()->success('Spedizione archiviata con successo')->push();

        //refresh the values inside powergrid table
        $this->closeModalWithEvents([
            'pg:eventRefresh-default',
        ]);
}
`

With this method, the toast notification(s) are displayed only if I refresh (or change) page. 
It is possibile to show notifications without refreshing the page?

Thank you in advance

**Comment by usernotnull** [comment]
I have not used that library, but before going further, have you tested with pushOnNextPage instead of push method?

**Comment by gallomaf** [comment]
I tried with pushOnNextPage but not work

**Reply by ashek1412** [reply]
Hi, is there any update
